Viewer/src/actionManager.ts

```diff
@@ -0,0 +1,40 @@
+import { EventCallback, TemplateManager } from "./templateManager";
+
+
+export class ActionManager {

```

Comment:
Do we want to use the same name as BABYLON.ActionManager?

Reply:
Noted. I will change it to EventManager maybe? since it is actually triggered by native js events

Reply:
Perfect!